server/summary_biplot_server.R

```diff
@@ -708,15 +742,11 @@ output$SUMMARY_BIPLOT <- renderPlotly({
         "No data available for \nthe selected variables"
     }
 
-    if (x_var == "Year" & y_var %in% c("Soil", "Geochemistry", "Hydrology", "Terrain")) {
-        empty_msg <- 'Non-temporal data selected. \nSet Aggregation to "Full record"'
-    }
-
-    if (x_var == "Year" & y_var %in% c("Soil", "Geochemistry", "Hydrology", "Terrain")) {
+    if (x_var == "Year" & y_var %in% c("Soil", "Geochemistry", "Hydrology", "Terrain", "Lithology")) {
         empty_msg <- 'Non-temporal data selected. \nSet Aggregation to "Full record"'
     }
 
-    empty_plot <- plotly::plot_ly() %>%
+    empty_plot <- plotly::plot_ly(type = "scatter") %>%
```

Comment:
@westonslaughter oh nice! i'm guessing this is for getting rid of that ubiquitous warning. thank you